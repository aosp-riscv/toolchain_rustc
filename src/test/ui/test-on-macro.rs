<<<<<<< HEAD   (086005 Importing rustc-1.38.0)
// build-pass (FIXME(62277): could be check-pass?)
// compile-flags:--test

#![deny(warnings)]

macro_rules! foo {
    () => (fn foo(){})
}

#[test]
foo!();

fn main(){}
=======
>>>>>>> BRANCH (8cd2c9 Importing rustc-1.39.0)
