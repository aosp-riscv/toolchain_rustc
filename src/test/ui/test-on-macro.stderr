<<<<<<< HEAD   (086005 Importing rustc-1.38.0)
warning: `#[test]` attribute should not be used on macros. Use `#[cfg(test)]` instead.
  --> $DIR/test-on-macro.rs:11:1
   |
LL | foo!();
   | ^^^^^^^

=======
>>>>>>> BRANCH (8cd2c9 Importing rustc-1.39.0)
